hadoop-pipeline-application/src/test/java/org/molgenis/hadoop/pipeline/application/inputdigestion/CommandLineInputParserTester.java

```diff
@@ -49,24 +49,20 @@
 	@BeforeClass
 	public void beforeClass() throws IOException
 	{
-		classLoader = Thread.currentThread().getContextClassLoader();
-		// System.out.println(classLoader.getResource("").toString());
-		// Output: <path/to/dir>/hadoop-pipeline/hadoop-pipeline-application/target/test-classes/
-
```

Comment:
Noted. It was intentially put there for easy reference when adding/adjusting test resource file locations needed by test classes, though I'll adjust it to a variable comment.